### ISSUE TITLE
`set_attribute` doesn't replace existing attributes

### ISSUE THREAD

**Issue by moogle19** [phoenixframework/phoenix_live_view#1868]
### Environment

* Elixir version (elixir -v): 
Erlang/OTP 24 [erts-12.2] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1]

Elixir 1.13.1 (compiled with Erlang/OTP 24)

* Phoenix version (mix deps): 1.6.6
* Phoenix LiveView version (mix deps): 0.17.6
* Operating system: macOS
* Browsers you attempted to reproduce this bug on (the more the merrier): Chrome
* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no: Yes

### Actual behavior

Calling `set_attribute` with an existing attribute doesn't replace it, e.g. following code doesn't change the value:

```
<input id="new_planner_action" name="new_planner[action]" type="hidden" value="save">
<button 
  phx-click={JS.set_attribute({"value", "publish"}, to: "#new_planner_action")}
>
  Set Attribute
</button>
```

Removing the attribute first works:
```
JS.remove_attribute("value", to: "#new_planner_action") 
|> JS.set_attribute({"value", "publish"}, to: "#new_planner_action")
```

### Expected behavior

The attribute should be replaced with the wanted value. 
If that is not intended there should at least be a notice in the documentation, that existing values don't get replaced.

**Comment by Narven** [comment]
Thanks for posting this here, I was going to do it :D

**Comment by jonatanklosko** [comment]
Closed via #1918 :)
